lib/block_view_stake.go

```diff
@@ -32,9 +32,17 @@ import (
 // TYPES: StakeEntry
 //
 
+type StakingRewardMethod = uint8
+
+const (
+	StakingRewardMethodPayToBalance StakingRewardMethod = iota
```

Comment:
I'd avoid using iota. If you're going to use it I think the proper way is to then not define the other numbers. So like this:
```
StakingRewardMethodPayToBalance StakingRewardMethod = iota
StakingRewardMethodRestake      StakingRewardMethod
StakingRewardMethodWhatever1      StakingRewardMethod
StakingRewardMethodWhatever2      StakingRewardMethod
```

Obviously I hate this because you can't immediately look at the code and know what the number is without, at minimum, help from an IDE.

So we instead just go 0, 1, 2, ... and avoid using iota entirely.

Reply:
Ah gotchu. I'll update to use explicit the values 0 and 1 then